telegram/message.py

```diff
@@ -339,8 +339,8 @@ def chat_id(self):
     @property
     def link(self):
         """:obj:`str`: Convenience property. If the chat of the message is not
-        a private chat, returns a t.me link of the message."""
-        if self.chat.type != Chat.PRIVATE:
+        a private chat or normal group, returns a t.me link of the message."""
+        if self.chat.type not in [Chat.PRIVATE, Chat.GROUP]:
             if self.chat.username:
                 to_link = self.chat.username
             else:
```

Comment:
You can shorten this if clause. if there is no username, now it has to be supergroup and we just need to skip the leading -100

Reply:
right